recipe/meta.yaml

```diff
@@ -26,51 +26,43 @@ requirements:
   build:
     - r-base
     - r-base64enc
-    - r-catools
     - r-evaluate >=0.8
     - r-htmltools >=0.3.5
     - r-jsonlite
     - r-knitr >=1.14
+    - r-mime
     - r-rprojroot
     - r-stringr >=1.2.0
     - r-yaml >=2.1.5
-    - xorg-libxrender 0.9.*
-    - xorg-libsm 1.2.*
-    - xorg-libxext 1.3.*
+
   run:
-    - pandoc >=1.12.3
     - r-base
     - r-base64enc
-    - r-catools
     - r-evaluate >=0.8
     - r-htmltools >=0.3.5
     - r-jsonlite
     - r-knitr >=1.14
+    - r-mime
     - r-rprojroot
     - r-stringr >=1.2.0
     - r-yaml >=2.1.5
-    - xorg-libxrender 0.9.*
```

Comment:
the xorg-deps probably needs also to stay, as well as a pinned `pandoc >=2.0,<3.0`

Reply:
Thanks -- a tip from previous experience I assume. I added both these